### PR TITLE
Some improvements

### LINKED ISSUE #21: Add random ID to filenames
A lot of our video conference is just automatically uploaded to Streams, and no one cares their title, so all titles are simply "Meeting in General". This will cause error if I try to specify more than one `--videoUrls`. Maybe you should append the video ID to the output filename, that is unique.

### PR DESCRIPTION
**Pull request by kylon**
QOL changes and improvements

Fixes #21

### CODE REVIEW

File: destreamer.ts [hunk 1]
```diff
@@ -16,9 +16,14 @@ import axios from 'axios';
  */
 
 const argv = yargs.options({
-    videoUrls: { type: 'array', alias: 'videourls', demandOption: true },
     username: { type: 'string', demandOption: false },
-    outputDirectory: { type: 'string', alias: 'outputdirectory', default: 'videos' },
+    outputDirectory: { type: 'string', alias: 'o', default: 'videos' },
+    videoUrls: {
+        alias: "u",
```
Comment (lukaarma) [inline destreamer.ts]: I would use -V as an alias here so that we can use -u for --username

File: destreamer.ts [hunk 2]
```diff
@@ -139,11 +144,20 @@ async function rentVideoForLater(videoUrls: string[], outputDirectory: string, u
...
+            `Video${videoUrls.indexOf(videoUrl)}` :
+            sanitized;
+
+        // Add date
+        title += ' - '+date;
 
-        title = (sanitize(title) == "") ? 
-            `Video${videoUrls.indexOf(videoUrl)}` : 
-            sanitize(title);
+        // Add random index to prevent unwanted file overwrite!
```
Comment (lukaarma) [inline destreamer.ts]: Good job adding the date and the unique identifier

File: destreamer.ts [hunk 3]
```diff
@@ -247,9 +269,36 @@ async function getVideoInfo(videoID: string, session: any) {
             return playbackUrl;
         });
 
-    return [title, hlsUrl];
+    return [title, date, hlsUrl];
 }
 
+function getVideoUrls() {
+    if (argv.videoUrls.length === 0)
```
Comment (lukaarma) [inline destreamer.ts]: maybe here we should trow an error if the length of the array is 0, we have no urls (even if i think that being a required arg it cannot be empty)

File: destreamer.ts (hunk 1 above)
Comment (kylon) [reply destreamer.ts]: Done, "u" is now to username, "V" urls

File: destreamer.ts (hunk 3 above)
Comment (kylon) [reply destreamer.ts]: You are right, removed :)

**Pull request merged**